configs/permissions/playbook-dispatcher.json

```diff
@@ -0,0 +1,10 @@
+{
+    "run": [
```

Comment:
@coderbydesign is the permission definition OK like this or do I also need to define the various permutations with wildcards?

Reply:
You won't have any system/canned roles with any of these permissions available, this is just for custom role creation? If so, these granular permissions should be all you need! Custom role creation doesn't use wildcards. 

I'll let @nicole-campbell12 correct me if I'm wrong here. 😄

Reply:
These permissions will be referenced from system/canned roles (e.g. the remediations role below, plus others) however neither of these needs wildcards.